### DISCUSSION TITLE
全局变量添加不上去

### DISCUSSION THREAD

**Discussion by Burninghamster** [illacloud/illa-builder#935] [Q&A]
1,怎么拿到另外一个查询里面返回的数据或者怎么设置全局变量
![1676513249180](https://user-images.githubusercontent.com/33534997/219249255-44302d86-0fdb-4acf-8fbd-445cb22d3885.png)
2：transformer怎么传参，

![1676513569241](https://user-images.githubusercontent.com/33534997/219250035-92038fd2-cd14-4f84-8f21-db2310463b72.png)
![1676513603422](https://user-images.githubusercontent.com/33534997/219250138-27cc7a60-2423-47b6-a10c-143f71ac1046.png)
谢谢

**Comment by AruSeito** [comment]
您好，
第一个问题：{{xxxx}}就可以作为全局变量使用。如图，看您图2的显示，推测是您的 mysql1.data 使用了 transform，所以最终结果是个1
![image](https://user-images.githubusercontent.com/41471697/219300000-82b9d9b6-ccfe-450d-9bb3-1227949a59ef.png)
第二个问题：transform 内不能传参，都是直接使用全局变量的。

**Reply by Burninghamster** [reply]
![1676539582842](https://user-images.githubusercontent.com/33534997/219323876-77b1abb9-38ec-46aa-bfb8-a3f63713c534.png)
图2是因为mysql1接口返回值是1，MySQL2我需要拿到mysql1返回的值，看着成功了，实际调取值的时候出现了参数缺少。
![1676539790853](https://user-images.githubusercontent.com/33534997/219324826-027ed7a2-d166-4f11-9393-ca46c7dbca94.png)
我知道这样写{{}}可以作为全局变量使用，但是我接口请求的时候拿不到这个全局变量值，你们官网展示的是自己设置的全局变量可以在左边展示，我的就不行，组件可以拿到全局变量的值，但是在接口那边{{}}这样写就不行

**Reply by Burninghamster** [reply]
![1676540205270](https://user-images.githubusercontent.com/33534997/219326741-9920b131-3c85-4e7b-9769-37dfbcbbf4c5.png)

**Reply by AruSeito** [reply]
您好，第一点刚才实际检查后发现确实存在问题，预计在这两天内发布修复。
第二点自定义全局变量，该功能目前还未上线，考虑到用户使用简便等问题，还在优化中

**Reply by Burninghamster** [reply]
其实你们直接尝试搭建一下一个简单带权限得后台管理系统，这样才能发现影响流程的bug，您好添加小图标功能什么时候上呀？？？这个功能虽然小但是确实属于必不可少的功能，其他的我感觉你们做的挺棒的。体验是不错的

**Reply by AruSeito** [reply]
> 其实你们直接尝试搭建一下一个简单带权限得后台管理系统，这样才能发现影响流程的bug，您好添加小图标功能什么时候上呀？？？这个功能虽然小但是确实属于必不可少的功能，其他的我感觉你们做的挺棒的。体验是不错的

您好，感谢您的建议，我们目前也在尝试更多的场景在我们内部落地。
小图标是指小图标的组件吗？该组件已经在 codeReview 阶段了，马上就会发布。

**Reply by Burninghamster** [reply]
![1676956666948](https://user-images.githubusercontent.com/33534997/220254178-b7417437-d538-4212-b309-68076b600931.png)
嗯嗯，这个应该是bug

**Reply by AruSeito** [reply]
您好，该 BUG 为已知 BUG，目前已修复，最近几天会发布。
